test/runtests.jl

```diff
@@ -33,3 +41,37 @@ end
     @info "Testing LightDFG Variable and Factor Subtypes"
     include("LightDFGSummaryTypes.jl")
 end
+
+
+if get(ENV, "IIF_TEST", "") == "true"
```

Comment:
This is important to keep

Reply:
okay, think @GearsAD was using internal Julia function to test if IIF is defined and then conditions on that.  Either way is good at this point.

Reply:
This allows for easier matrix setup in travis.

Reply:
just din't want a merge conflict to revert it back.

Reply:
Yeah the Pkg check works a little cleaner than an environment variable but merging as is for now.

Reply:
Maybe i should rephrase. It’s important to keep the way i changed the travis setup. It fixed 4 things, so as long as they stay the same i don’t mind if iif is looked for. Fixed:
- iif test was done on latest tag
- jobs are now run in parallel 
- failures allowed to not break on circular dependancies 
-coverage report uploaded on pass

Reply:
@Affie, i'm all good with what you listed here.  So dont have to fiddle with that.